### PR TITLE
Doesnt work

### PR DESCRIPTION
**Pull request by CasperStaahl**

### CODE REVIEW

File: frontend/Pages/PosterScreen.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+.active {
+  background-color: #717171;
+}
+
+
+/* On smaller screens, decrease text size */
+@media only screen and (max-width: 300px) {
+  .text {font-size: 11px}
+}
+</style>
```
Comment (MadsBalslev) [inline frontend/Pages/PosterScreen.html:27]: Not correct indentation...

File: frontend/Pages/PosterScreen.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  var slides = document.getElementsByClassName("mySlides");
+  for (i = 0; i < slides.length; i++) {
+    slides[i].style.display = "none"; 
+  }
+  slideIndex++;
+  if (slideIndex > slides.length) {slideIndex = 1}   
+  slides[slideIndex-1].style.display = "block"; 
+  setTimeout(showSlides, 4000); // Change image every 2 seconds
+}
+</script>
```
Comment (MadsBalslev) [inline frontend/Pages/PosterScreen.html:62]: Not correct indentation

File: frontend/Pages/PosterScreen.razor [hunk 3]
```diff
@@ -3,82 +3,113 @@
...
+
+    <!-- <div class="loading" hidden=@hidetext>
         <h1> Loading picture number: @currentPicture ...</h1>
-    </div>
-    <div class="imgContainer" hidden=@hidepicture>
-        <img class="poster" src="@imageSource">
+    </div> -->
+
+    <div class="imgContainer" <img class="poster" src="@currentPicture">
     </div>
```
Comment (MadsBalslev) [inline frontend/Pages/PosterScreen.razor:18]: Something seems wrong here...

File: frontend/Pages/PosterScreen.razor [hunk 4]
```diff
@@ -3,82 +3,113 @@
...
-    <div class="loading" hidden=@hidetext>
+
+    <!-- <div class="loading" hidden=@hidetext>
         <h1> Loading picture number: @currentPicture ...</h1>
-    </div>
-    <div class="imgContainer" hidden=@hidepicture>
-        <img class="poster" src="@imageSource">
+    </div> -->
+
+    <div class="imgContainer" <img class="poster" src="@currentPicture">
```
Comment (MadsBalslev) [inline frontend/Pages/PosterScreen.razor:17]: This should be `<div class="imgContainer"></div>

**Review by MadsBalslev — CHANGES_REQUESTED** [review]

**Review by Mightyhaha — COMMENTED** [review]
DO NOT COMMIT/MERGE. Please.

**Comment by MadsBalslev** [comment]
Nicolai should've closed this

**Pull request closed without merging**